QEfficient/utils/generate_inputs.py

```diff
@@ -100,43 +95,29 @@ def prepare_ort_inputs(self, n_layer, padding_shape):
         :return inputs: Dict - input_ids, position_ids, past_key_values
         """
 
-        model_inputs = self.tokenizer(
+        inputs = self.tokenizer(
             self.input_str,
             return_tensors="np",
             padding=True,
         )
-        input_ids = model_inputs["input_ids"]
-
-        inputs = {}
-        inputs["input_ids"] = input_ids
-
-        batch_size, input_len = inputs["input_ids"].shape
+        input_ids = inputs["input_ids"]
+        batch_size, input_len = input_ids.shape
+        inputs.pop("attention_mask")
         position_ids = np.arange(input_len).reshape(1, -1)
-        print(batch_size, input_len, position_ids)
-
-        if len(input_ids.shape) == 1:
-            inputs["input_ids"] = input_ids.astype(np.int64)[:, np.newaxis]
-        else:
-            input_ids = np.concatenate(
-                [
-                    input_ids,
-                    np.ones((batch_size, self.prompt_len - input_len)) * (self.tokenizer.pad_token_id),
-                ],
-                axis=1,
-            ).astype(np.int64)
-            inputs["input_ids"] = input_ids.astype(np.int64)
-
-        if len(position_ids.shape) == 1:
-            inputs["position_ids"] = position_ids.astype(np.int64)[:, np.newaxis]
-        else:
-            position_ids = np.concatenate(
-                [
-                    position_ids,
-                    np.ones((batch_size, self.prompt_len - input_len)) * (-1),
-                ],
-                axis=1,
-            ).astype(np.int64)
-            inputs["position_ids"] = position_ids.astype(np.int64)
+        inputs["input_ids"] = np.concatenate(
+            [
+                input_ids,
+                np.ones((batch_size, self.prompt_len - input_len)) * (self.tokenizer.pad_token_id),

```

Comment:
You could use `np.full()` instead of `np.ones() * pad_token_id`, same below in line 117

Reply:
sure.